### PR TITLE
Minor tweaks to docs and naming

### LINKED ISSUE #12: Standardize "mission_method" usage throughout
### Summary of Issue

Within Aviary, there is a notion of `mission_method` as well as `mission_origin` which means the same thing.
There might be others cases or words describing the same thing.
To resolve this issue, use the same name throughout.

### Issue Type

- [ ] Bug
- [ ] Enhancement
- [x] Code Cleanup
- [ ] Docs
- [ ] Miscellaneous

### Is your feature request related to a problem? Please describe.

_No response_

### Associated Bug Report

_No response_

### LINKED ISSUE #21: Improve top-level docs organization
### Desired capability or behavior.

There is the table of contents widget in the upper right, which looks handy. But when you click on those links (eg, User Guide) , you go to a paragraph about the User Guide, not the links below for the user guide. And that paragraph doesn't even have a link to the section with the links.

We should add a table of contents to the top level pages for each section.

### Is your feature request related to a problem? Please describe.

_No response_

### Associated Bug Report

_No response_

### PR DESCRIPTION
**Pull request by johnjasa**
### Summary

Minor changes to:
- doc intro page for clarity based on user feedback
- combining `mission_method` and `mission_origin` into `mission_method`

### Related Issues

- Resolves #12 
- Resolves #21

### Backwards incompatibilities

None

### New Dependencies

None

### CODE REVIEW

**Review by Kenneth-T-Moore — APPROVED** [review]

File: aviary/interface/methods_for_level2.py [hunk 1]
```diff
@@ -226,14 +226,6 @@ def load_inputs(self, input_filename, engine_builder=None):
             aviary_inputs.set_val(Mission.Summary.GROSS_MASS,
                                   val=self.initial_guesses['actual_takeoff_mass'], units='lbm')
 
-            # Commonly referenced values
-            self.enable_engine_sizing = aviary_inputs.get_val(
-                Aircraft.Engine.SCALE_MASS)
-            self.cruise_alt = aviary_inputs.get_val(
-                Mission.Design.CRUISE_ALTITUDE, units='ft')
-            self.problem_type = aviary_inputs.get_val('problem_type')
```
[the remark this answers is not in the document]
Comment (crecine) [reply aviary/interface/methods_for_level2.py:234]: cruise_alt and problem_type are used elsewhere, these should not be removed

**Review by crecine — CHANGES_REQUESTED** [review]
cruise_alt and problem_type are used elsewhere

File: aviary/interface/methods_for_level2.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (johnjasa) [reply aviary/interface/methods_for_level2.py:234]: For both of these the values are later overwritten; the values saved in `load_inputs` are replaced by values read in in `_add_gasp_takeoff_systems` and `add_design_variables`. I believe it is fine to be removed from `load_inputs` based on this. Do you see a reason to keep these value instantiations as they stand?

[the remark this answers is not in the document]
Comment (crecine) [reply aviary/interface/methods_for_level2.py:234]: as long as that happens before the phases are added to the problem, that should be fine

**Review by crecine — APPROVED** [review]

**Pull request merged**